server/src/test/java/org/apache/druid/server/coordinator/DruidCoordinatorBalancerTest.java

```diff
@@ -656,11 +659,19 @@ private DruidCoordinatorRuntimeParams setupParamsForMaintenancePriority(int prio
     )
         .withDynamicConfigs(
             CoordinatorDynamicConfig.builder()
-                                    .withMaxSegmentsToMove(
-                                        1
-                                    ).withMaintenanceModeSegmentsPriority(priority).build()
+                                    .withMaxSegmentsToMove(1)
+                                    .withMaintenanceModeSegmentsPriority(priority)
+                                    .build()
         )
         .withBalancerStrategy(strategy)
         .build();
   }
+
+  private static ImmutableMap<String, DataSegment> segmentsToMap(DataSegment... segments)
+  {
+    return ImmutableMap.copyOf(
```

Comment:
Could be `Maps.uniqueIndex(Arrays.asList(segments), DataSegment::getIdentifier)`

Reply:
✅